tests/integration/test_data/yarn_e2e_test_multiple_packages/container/Containerfile

```diff
@@ -0,0 +1,25 @@
+FROM docker.io/node:18.19.0
+
+# Test disabled network access
+RUN if curl -IsS www.google.com; then echo "Has network access!"; exit 1; fi
+
+# Print cachi2 env vars file
+RUN cat /tmp/cachi2.env
+
+# Check yarn deps
+RUN ls /tmp/yarn_e2e_test_multiple_packages-output/deps/yarn/cache
+
+# Check content of source repository folder
+RUN ls /tmp/yarn_e2e_test_multiple_packages-source
+
+WORKDIR /tmp/yarn_e2e_test_multiple_packages-source/third-pkg
+RUN . /tmp/cachi2.env && yarn install
+
+WORKDIR /tmp/yarn_e2e_test_multiple_packages-source/second-pkg
+RUN . /tmp/cachi2.env && yarn install
+
+WORKDIR /tmp/yarn_e2e_test_multiple_packages-source/first-pkg
+RUN . /tmp/cachi2.env && yarn install
+
+RUN cp -r /tmp/yarn_e2e_test_multiple_packages-source/first-pkg /src
+WORKDIR /src
```

Comment:
If I comment out these two lines, the test will fail. We encounter the same problem in main (regular installs) e2e test [here](https://github.com/containerbuildsystem/cachi2/blob/c2e5f5968df6a57f59fa3e383ff876251c3e46e2/tests/integration/test_data/yarn_e2e_test/container/Containerfile#L16)